.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
   - id: black
     args: [--config=pyproject.toml]
 - repo: https://github.com/timothycrosley/isort
-  rev: 5.10.1
+  rev: 5.12.0
```

Comment:
I updated the pre-commit hooks on main, you can drop these changes from the PR. If you need these updated versions you can now rebase from main

Reply:
ok!